main.go

```diff
@@ -15,54 +15,49 @@
 package main
 
 import (
-	"flag"
 	"fmt"
-	"net"
 	"os"
+	"time"
 
-	"github.com/sirupsen/logrus"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/grpclog"
-	"google.golang.org/grpc/reflection"
-
+	"github.com/layer5io/gokit/errors"
+	"github.com/layer5io/gokit/logger"
+	"github.com/layer5io/gokit/utils"
+	"github.com/layer5io/meshery-adapter-library/api/grpc"
 	"github.com/layer5io/meshery-consul/consul"
-	mesh "github.com/layer5io/meshery-consul/meshes"
+	"github.com/layer5io/meshery-consul/internal/config"
+	"github.com/layer5io/meshery-consul/internal/operations"
 )
 
 var (
-	gRPCPort = flag.Int("grpc-port", 10002, "The gRPC server port")
+	serviceName    = "consul-adaptor"
```

Comment:
Declare a `const` instead of `var` Unless the `serviceName` Can change??

Reply:
Agreed. I'll fix that.

Reply:
s/adaptor/adapter/g

Reply:
s/wish/.../g